internal/quickstart/container.go

```diff
@@ -112,6 +112,7 @@ func (m ContainerModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		m.flagKey = msg.flag.key
 		m.currentStep += 1
 		m.err = nil
+		cmd = sendEnableMouseCellMotionMsg()
```

Comment:
we can either enable this here as a message or at the start of the quickstart program, thought it was safer to enable here?

Reply:
I think it'd better here too. Since we don't use the feature elsewhere